src/driver.rs

```diff
@@ -123,6 +123,8 @@ impl Driver {
                 options_c as *mut *mut i8,
             )
         };
+        unsafe { gdal_sys::CSLDestroy(options_c) };
```

Comment:
On line 110, there are loops over `options_c`. Is this pointer just re-used in 123? Otherwise, we would only destroy the last return of `CSLSetNameValue` if we would have more than one option.
And if we have no option, we would destroy `null_mut`?

Reply:
Frees the passed string list (null terminated array of strings). It is safe to pass NULL.
https://gdal.org/doxygen/ogr__api_8h.html#a4000d426bf26ad7cc7d4012634c93f09

Reply:
it is also used in gdal this way: https://github.com/OSGeo/gdal/blob/d0a12d1a74e08557605f57cf2d94dac82b9fdda9/gdal/apps/gdalwarp_lib.cpp#L664

Reply:
Seems okay to me, I somewhat can't resolve this comment.